e2e/suite/job_generate_test.go

```diff
@@ -6,12 +6,12 @@ import (
 	"io"
 	"time"
 
+	. "github.com/onsi/ginkgo/v2" //nolint:revive
```

Comment:
Can you perhaps disable whatever check is failing in the `.golangci.json` instead of adding nolint comments to the imports?